stroyprombeton/tests/tests_selenium.py

```diff
@@ -1,18 +1,43 @@
+import unittest
+
 from django.core import mail
 from django.db.models import Count
 from django.template.defaultfilters import floatformat
 from django.urls import reverse
 from selenium.webdriver.common.action_chains import ActionChains
 from selenium.webdriver.common.keys import Keys
 from selenium.webdriver.common.by import By
-from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support import expected_conditions as EC, ui
 
 from pages.models import Page
 
-from stroyprombeton.models import Category, Product
+# @todo #320:15m - Do the whole `models` import in tests_selenium
```

Comment:
Import the `stroyprombeton.models` package as `stb_models` for the whole project.

Reply:
@artemiy312 , it's too long. I suggest this:
`Import `models` to it's own var`